### PR TITLE
Elaborate the password change process

### PR DESCRIPTION
**Pull request by jcscottiii**
Per https://favro.com/card/1e11108a2da81e3bd7153a7a/18F-5602
and
https://cloud-gov.zendesk.com/agent/tickets/523

https://gsa-tts.slack.com/archives/G3JNU3HJL/p1496422054576583

### CODE REVIEW

File: uaaextras/templates/email/body-expiring-password.html [hunk 1]
```diff
@@ -11,7 +11,8 @@
             will expire in {{password.daysUntilExpiration}} days.
         </p>
         <p>
-            Please <a href="{{password.loginLink}}">log in</a> with your current password</a> and then <a href="{{password.changeLink}}">change your password</a>.
+            First, <a href="{{password.loginLink}}">log in</a> with your current password.
+            After you login, use this <a href="{{password.changeLink}}">link</a> to change your password.
```
Comment (LinuxBozo) [inline uaaextras/templates/email/body-expiring-password.html]: Using link text phrases like "this link" usually is considered a no-no in accessibility.

Comment (jcscottiii) [inline uaaextras/templates/email/body-expiring-password.html]: hmm. i wonder what alternatives we could do to make sure they know there is a second step? (Do a numbered list?)

Comment (LinuxBozo) [inline uaaextras/templates/email/body-expiring-password.html]: I think a list of some type (bulleted, numbered, doesn't really matter) would work better to let folks know there's a process to follow, yeah.

Comment (brittag) [inline uaaextras/templates/email/body-expiring-password.html]: I took a shot at updating the link text to make it reflect the content of the link.

**Pull request merged**